core/embed/rust/src/ui/model_tr/component/input_methods/choice.rs

```diff
@@ -37,9 +40,10 @@ pub trait Choice<T: StringType> {
 /// This way, no more than one item is stored in memory at any time.
 pub trait ChoiceFactory<T: StringType> {
     type Action;
+    type Item: Choice<T>;
```

Comment:
is this ever something other than `ChoiceItem`?

Reply:
`MenuChoice` in `bootloader/menu.rs`